languages/kotlin/src/main/cpp/jni_view_node.cpp

```
@@ -124,6 +124,30 @@ JNIEXPORT void JNICALL Java_com_github_kavos113_karin_engine_jni_JniViewNode_set
     }
 }
 
+JNIEXPORT void JNICALL Java_com_github_kavos113_karin_engine_jni_JniViewNode_setMarginSide
+    (JNIEnv *env, jclass cls, jlong viewPtr, jchar side, jfloat margin)
+{
+    CHECK_JNI_PTR(viewPtr);
```

Cppcheck (2.20.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)